docs/en/stack/ml/df-analytics/dfanalytics-limitations.asciidoc

```diff
@@ -135,12 +127,12 @@ performance profile.
 [[dfa-feature-importance-limitation]]
 === Analytics runtime performance may significantly slow down with feature importance computation
 
-For complex models – those with many deep trees – the calculation of 
-feature importance takes significant extra time. Feature importance is calculated at the end 
-of the analysis and therefore the job may appear to be stuck at 99% for several 
-hours.
+For complex models – those with many deep trees – the calculation of feature 
```

Comment:
Unless this is the only type of model that is complex, I'd recommend this change:
```suggestion
For complex models (such as those with many deep trees), the calculation of feature 
```